packages/x-charts-pro/src/ChartContainerPro/ChartContainerPro.tsx

```diff
@@ -41,9 +41,9 @@ const ChartContainerPro = React.forwardRef(function ChartContainer(
   return (
     <DrawingProvider {...drawingProviderProps}>
       <PluginProvider {...pluginProviderProps}>
-        <SeriesProvider {...seriesProviderProps}>
-          <ZoomProvider {...zoomProviderProps}>
-            <CartesianContextProviderPro {...cartesianContextProps}>
+        <ZoomProvider {...zoomProviderProps}>
+          <SeriesProvider {...seriesProviderProps}>
+            <CartesianProviderPro {...cartesianProviderProps}>
```

Comment:
I assume the reordeing has a reason in the PR for zoom filterMode?

Reply:
yep, though I think in the end I didn't use it

Reply:
Maybe this can be reverted then to avoid introducing modification. But that recommendation seems more like psychologic safety than true engineering 😇